ports/qyoblaster/Q-YO Blaster.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+XDG_DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+elif [ -d "$XDG_DATA_HOME/PortMaster/" ]; then
+  controlfolder="$XDG_DATA_HOME/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+get_controls
+
+# Variables
+GAMEDIR="/$directory/ports/qyoblaster"
+
+# CD and set permissions
+cd $GAMEDIR
+> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1
+$ESUDO chmod +x -R $GAMEDIR/*
```

Comment:
When possible use a fine permission set
```suggestion
$ESUDO chmod +x $GAMEDIR/gmloadernext
$ESUDO chmod +x $GAMEDIR/tools/SDL_swap_gpbuttons.py
$ESUDO chmod +x $GAMEDIR/tools/gmKtool.py
$ESUDO chmod +x $GAMEDIR/tools/oggdec
$ESUDO chmod +x $GAMEDIR/tools/oggenc
$ESUDO chmod +x $GAMEDIR/tools/patchscript
$ESUDO chmod +x $GAMEDIR/tools/xdelta3

Reply:
you may need to add
`$ESUDO chmod +x $GAMEDIR/tools/splash`